src/views/registration-js/content.php

```diff
@@ -5,24 +5,30 @@
  * Override this template in your own theme by creating a file at:
  * [your-theme]/tribe/tickets/registration-js/content.php
  *
- * @since 4.11.0
- * @since 4.11.3.1 Fix handling where $provider is an object.
- * @since 4.12.0 Prevent potential errors when $provider_obj is not valid.
- * @since 4.12.1    Set the notice as hidden by default. The JavaScript will show it if needed.
+ * @since   4.11.0
+ * @since   4.11.3.1 Fix handling where $provider is an object.
+ * @since   4.12.0   Prevent potential errors when $provider_obj is not valid.
+ * @since   4.12.1   Set the notice as hidden by default. The JavaScript will show if needed.
+ * @since   TBD Update for getting ticket provider now returning instance or False. Remove duplicate array property.
+ *              Retrieve $is_meta_up_to_date in a manner consistent with other template variables. Moved `novalidate` from
+ *              div to form, as it used to be. Array short syntax.
  *
- * @version 4.12.0
+ * @version TBD
+ *
+ * @var Tribe__Tickets__Attendee_Registration__View $this
  */
 $provider = $this->get( 'provider' ) ?: tribe_get_request_var( 'provider' );
-$events = $this->get( 'events' );
+$events = (array) $this->get( 'events' );
```

Comment:
:warning: Equals sign not aligned with surrounding assignments; expected 3 spaces but found 1 space

<sub>hash: ad8eb6a00e663247816c14a7872381f9</sub>

Reply:
fixed